imagersite/imagerprofile/handlers.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
+from django.dispatch import receiver
+from django.db.models.signals import post_save, post_delete
+from django.contrib.auth.models import User
+from .models import ImagerProfile
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, **kwargs):
+    instance = kwargs.get('instance')
+    if not instance or kwargs.get('raw', False):
+        return
+    try:
+        instance.profile
+    except:
```

Comment:
Bare excepts are bad.  Find out the error that should be raised here and catch it specifically.